org.zrythm.Zrythm.json

```diff
@@ -0,0 +1,442 @@
+            "build-options": {
+                "env": {
+                    "Vamp_CFLAGS": " ",
+                    "Vamp_LIBS": " "
+                }
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://code.breakfastquay.com/attachments/download/34/rubberband-1.8.1.tar.bz2",
+                    "sha256": "ff0c63b0b5ce41f937a8a3bc560f27918c5fe0b90c6bc1cb70829b86ada82b75"
+                }
+            ],
+            "post-install": [
+                "install -Dm644 -t /app/share/licenses/rubberband COPYING"
+            ],
+            "cleanup": [
+                "/bin",
+                "/include",
+                "/share/ladspa",
+                "*.a",
+                "*.so"
+            ]
+        },
+        {
+            "name": "reproc",
+            "buildsystem": "cmake",
+            "build-options": {
+                "env": {
+                    "BUILD_SHARED_LIBS": "ON",
+                    "REPROC_WARNINGS": "ON",
+                    "REPROC_WARNINGS_AS_ERRORS": "ON",
+                    "CFLAGS": "-fPIC"
+                }
+            },
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/DaanDeMeyer/reproc",
+                    "commit": "8aea3aef64c37a7ed0db759a7128a80c2540c808",
+                    "tag": "v14.2.4"
+                }
+            ]
+        },
+        {
+            "name": "breeze-icons",
+            "builddir": true,
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/icons/breeze-dark",
+                "cp -r icons-dark/* /app/share/icons/breeze-dark"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/KDE/breeze-icons",
+                    "commit": "c8ca52623d49f77fe32dc1bdb400fa16bc4afb06",
+                    "tag": "v5.90.0"
+                }
+            ]
+        },
+        {
+            "name": "zrythm",
+            "builddir": true,
+            "buildsystem": "meson",
+            "config-opts": [
+                "-Dfor_flathub=true"
+            ],
+            "build-options": {
+                "env": {
+                    "LIBRARY_PATH": "/app/lib"
+                }
+            },
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://git.sr.ht/~alextee/zrythm",
+                    "commit": "f28c06672078ff8f1be16b2abd2b9ab2560f189f",
+                    "tag": "v1.0.0-alpha.28.1.3",
+                    "x-checker-data": {
+                        "type": "git",
+                        "tag-pattern": "^v(.*[\\d.]+)$"
+                    }
+                }
+            ],
+             "post-install": [
+                "mkdir -pm755 $FLATPAK_DEST/extensions/Plugins"
```

Comment:
we usuall use `install -d` and post-install in commonl found above `sources`

Reply:
done locally